docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -177,12 +167,11 @@ If document submission is supported, navigate to **Settings > Setup > Tax > _Tax
 
 Prior to testing a tax provider, the merchant or partner test store should have the following configured in the control panel:
 
-* The store default country, found by navigating to **Settings > Setup > Store profile**, is set to a country that is supported by the tax provider
```

Comment:
We are removing this requirement from the flow now.

That is, even if the store is based in Australia the store will be able to see TaxJar (United States only tax provider) and select it to be enabled for United States transactions.

That is, we don't want to get in the way of offering flexible international selling configuration.